src/Ledger/PParams.lagda

```diff
@@ -41,49 +41,40 @@ record PParams : Set where
 \emph{Network group}
 \AgdaTarget{maxBlockSize, maxTxSize, maxHeaderSize, maxValSize, pv}
 \begin{code}
-        maxBlockSize       : ℕ
-        maxTxSize          : ℕ
-        maxHeaderSize      : ℕ
-        maxValSize         : ℕ
-        pv                 : ProtVer -- retired, keep for now
+        maxBlockSize maxTxSize maxHeaderSize maxValSize  : ℕ
```

Comment:
This style is more compact and is consistent with other (non-documented) code in this module.  I suggest we adopt this style for all documented records and datatypes.

Reply:
I agree that in most cases it makes sense, but there might be cases where it is nicer to put distinct fields of the same type in distinct lines.

Also, when using such syntax you shouldn't vertically align on the colon `:`, it is too much rebasing overhead for an association that is confusing to begin with.

Reply:
Okay, I'll change it back.